app/models/orchestration_stack.rb

```diff
@@ -85,6 +85,22 @@ def raw_update_stack(_template, _options = {})
     raise NotImplementedError, _("raw_update_stack must be implemented in a subclass")
   end
 
+  def update_stack_queue(userid, template, options = {})
```

Comment:
Where does this method get called? 
If we add queue version to update, shouldn't we do the same to create and delete?